Source/SwiftLintCore/Protocols/SwiftSyntaxCorrectableRule.swift

```diff
@@ -19,8 +19,7 @@ public extension SwiftSyntaxCorrectableRule {
             return []
         }
 
-        file.write(newTree.description)
-        return rewriter
+        let corrections = rewriter
```

Comment:
This change was necessary because given an input like

```
internal struct Foo {
    var bar: String

  ↓internal init(bar: String) {
        self.bar = bar
    }
}
```

and a correction like

```
internal struct Foo {
    var bar: String
}
```

The violation location is out of bounds in the corrected code. We now resolve the correction location in the original code, and then write out the corrected code.